OpenSourcePolitics/decidim-module-blog_author_petition
# fix multiples details

This PR : 
- removes the organization from the author selection field when creating a post as a petition's author.
- It removes the author field if no user groups, to replace it with a hidden field.
- It authorizes the user to modify only his posts
- It adds warning message when no posts are published
- It adds warning message when the initiative is validating
- It removes the creation button when the initiative or the component are not published

How to test :
- Create an initiative
- Publish it
- Create post
- See you can't select organization
- Go to admin dashboard and try to create post in the same component
- See you can select organization

- Go to the posts
- Remove every post
- See the warning message
- Send the initiative to technical validation
- Go back to posts
- See the warning message and see you can't create posts
